move/seal-policy/sources/seal_policy.move

```diff
@@ -0,0 +1,31 @@
+// Copyright (c), Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module seal_policy_example::seal_policy {
+    use enclave::enclave::Enclave;
+    use seal_policy_example::weather::WEATHER;
+    use sui::hash::blake2b256;
+
+    const ENoAccess: u64 = 0;
+
+    entry fun seal_approve(_id: vector<u8>, enclave: &Enclave<WEATHER>, ctx: &TxContext) {
```

Comment:
assert id = 0

Reply:
multiple enclaves to provision this without seal. can add more restrictions to latest version etc